src/components/Charts/RowChart/RowChart.js

```diff
@@ -0,0 +1,272 @@
+import { scrollToFocus } from '../../../utils/trends';
+import {
+  selectQueryLens,
+  selectQueryTab,
+} from '../../../reducers/query/selectors';
+import { selectAggsState } from '../../../reducers/aggs/selectors';
+import {
+  selectViewIsPrintMode,
+  selectViewExpandedRows,
+} from '../../../reducers/view/selectors';
+import {
+  cloneDeep,
+  coalesce,
+  getAllFilters,
+  sendAnalyticsEvent,
+} from '../../../utils';
+import { MODE_MAP } from '../../../constants';
+
+export const RowChart = ({
+  helperText,
+  id,
+  colorScheme,
+  data,
+  title,
+  total,
+}) => {
+  const dispatch = useDispatch();
+  const tab = useSelector(selectQueryTab);
+  const queryLens = useSelector(selectQueryLens);
+  const aggs = useSelector(selectAggsState);
+  const expandedRows = useSelector(selectViewExpandedRows);
+  const isPrintMode = useSelector(selectViewIsPrintMode);
+  //const width = useSelector(selectViewWidth);
+  const lens = tab === MODE_MAP ? 'Product' : queryLens;
+
+  useEffect(() => {
+    const chartID = '#row-chart-' + id;
+
+    const formatTip = (value) => {
+      return value.toLocaleString() + ' complaints';
+    };
+
+    const wrapText = (text, width, viewMore) => {
+      // ignore test coverage since this is code borrowed from d3 mbostock
+      // text wrapping functions
+      /* eslint-disable complexity */
+      /* istanbul ignore next */
+      text.each(function () {
+        const innerText = d3.select(this);
+        const spanWidth = viewMore ? innerText.attr('x') : 0;
+        if (innerText.node().children && innerText.node().children.length > 0) {
+          // assuming its already split up
+          return;
+        }
+        const words = innerText.text().split(/\s+/).reverse(),
+          // ems
+          lineHeight = 1.1,
+          // eslint-disable-next-line id-length
+          y = innerText.attr('y') || 0,
+          dy = parseFloat(innerText.attr('dy') || 0);
+
+        let word,
+          line = [],
+          lineNumber = 0,
+          wrapCount = 0,
+          tspan = innerText
+            .text(null)
+            .append('tspan')
+            .attr('x', spanWidth)
+            .attr('y', y)
+            .attr('dy', dy + 'em');
+
+        // eslint-disable-next-line no-cond-assign
+        while ((word = words.pop())) {
+          line.push(word);
+          tspan.text(line.join(' '));
+          if (tspan.node().getComputedTextLength() > width) {
+            line.pop();
+            tspan.text(line.join(' '));
+            line = [word];
+            tspan = innerText
+              .append('tspan')
+              .attr('x', spanWidth)
+              .attr('y', y)
+              // eslint-disable-next-line no-mixed-operators
+              .attr('dy', ++lineNumber * lineHeight + dy + 'em')
+              .text(word);
+            wrapCount++;
+          }
+        }
+
+        // only allow this to go through if not IE
+        if (wrapCount && !window.document.documentMode) {
```

Comment:
Wow
it's been a while since i looked at this, but we dropped IE support a long time ago.

This should be changed to 
`if (wrapCount) {`

Reply:
Oh good catch, didn't even realize that. This has been taken out.